engine/src/main/java/org/pentaho/di/pan/PanCommandExecutor.java

```diff
@@ -150,8 +152,8 @@ public Result execute( final Params params, String[] arguments ) throws Throwabl
         // Try to load the transformation from file, even if it failed to load from the repository
         // You could implement some fail-over mechanism this way.
         if ( trans == null ) {
-          trans = loadTransFromFilesystem( params.getLocalInitialDir(), params.getLocalFile(), params.getLocalJarFile(),
-            params.getBase64Zip() );
+          trans = loadTransFromFilesystem( DefaultBowl.getInstance(), params.getLocalInitialDir(),
```

Comment:
This can potentially create problems. In the Pan/Kitchen.java, we are taking the project parameter to figure out the what is the ProjctBowl. This project bowl will be set in the TransMeta/JobMeta when the loadXml is called. But here, we are initializing TransMeta with the defaultBowl.